.vscode/settings.json

```diff
@@ -0,0 +1,18 @@
+{
+    "files.exclude": {
```

Comment:
On second thought, I would prefer if the files were not hidden. They can be made hidden on your side only if you want. The user configuration files shouldn't be in the source control imo.

Reply:
Should I add the file to the `.gitignore`?

Reply:
You can add it